server/front/src/index.ts

```diff
@@ -274,32 +274,80 @@ export function start (
   // eslint-disable-next-line @typescript-eslint/no-misused-promises
   app.get('/files/*', filesHandler)
 
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Could you please remove comments, since it will not be required anymore. Thanks.